de.dlr.sc.overtarget.language.ui/META-INF/MANIFEST.MF

```diff
@@ -30,6 +30,7 @@ Require-Bundle: de.dlr.sc.overtarget.language,
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Export-Package: de.dlr.sc.overtarget.language.ui,
  de.dlr.sc.overtarget.language.ui.contentassist,
+ de.dlr.sc.overtarget.language.ui.editor,

```

Comment:
My editor shows the warning, that it cannot export this package because it doesn't exist... What class shoud be in there?